src/tags/tests/test_models.py

```diff
@@ -179,6 +179,30 @@ def test_multiple_tags_per_quest_badges(self):
 
         self.assertEqual(expected_order, calculated_order)
 
+    def test_query_correct_quest(self):
```

Comment:
test naming convention.

Reply:
calling the function to ```test_query_tag_from_approved_quest``` im not specifying the function being used as ```Tag_get_user_tags_and_xp_Tests``` says "Specialized TestClass for testing get_user_tags_and_xp function"